streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -233,44 +234,7 @@ public void closeCleanAndRecycleState() {
         closeTaskSensor.record();
         transitionTo(State.CLOSED);
 
-        log.info("Closed clean and recycled state");
-    }
-
-    /**
-     * Create an active task from this standby task without closing and re-initializing the state stores.
-     * The task should have been in suspended state when calling this function
-     *
-     * TODO: we should be able to not need the input partitions as input param in future but always reuse
-     *       the task's input partitions when we have fixed partitions -> tasks mapping
-     */
-    public StreamTask recycle(final Time time,
-                              final ThreadCache cache,
-                              final RecordCollector recordCollector,
-                              final Set<TopicPartition> inputPartitions,
-                              final Consumer<byte[], byte[]> mainConsumer) {
-        if (!inputPartitions.equals(this.inputPartitions)) {
-            log.warn("Detected unmatched input partitions for task {} when recycling it from active to standby", id);
-        }
-
-        stateMgr.transitionTaskType(TaskType.ACTIVE);
-
-        log.debug("Recycling standby task {} to active", id);
-
-        return new StreamTask(
-            id,
-            inputPartitions,
-            topology,
-            mainConsumer,
-            config,
-            streamsMetrics,
-            stateDirectory,
-            cache,
-            time,
-            stateMgr,
-            recordCollector,
-            processorContext,
-            logContext
-        );
+        log.info("Closed and recycled state, and converted type to active");
```

Comment:
```suggestion
        log.info("Closed and recycled state");
```
Conversion is now done in `createActiveTaskFromStandby()`.

Reply:
ack.